cl/alerts/templates/docket_alert_email.html

```diff
@@ -60,6 +60,28 @@ <h3 class="alt bottom" style="font-size: 1.5em; font-weight: normal; line-height
 
     <hr style="background: #ddd; color: #ddd; clear: both; float: none; width: 60%; height: .1em; margin: 0 0 1.45em; border: none;">
 
+    {% if notes or tags %}
+      <p style="font-weight: inherit; font-family: inherit; border: 0; vertical-align: baseline; font-style: inherit; margin: 1em 0 1em; padding: 0;">
+        <strong>Your Note: </strong><span>{% if notes %}{{ notes }}{% else %}You have not added notes to this case. <a href="https://www.courtlistener.com{% url 'tag_favorites_help' %}">Learn more</a>{% endif %}</span>
+      </p>
+      <p style="font-weight: inherit; font-family: inherit; border: 0; vertical-align: baseline; font-style: inherit; margin: 1em 0 1em; padding: 0;">
+        <strong>Your Tags: </strong>
+        <span>
+          {% if tags %}
+            {% for tag in tags %}
+              <a href="https://www.courtlistener.com{% url 'view_tag' username tag.name %}">{{ tag.name }}</a>{% if not forloop.last %}, {% endif %}
+            {% endfor %}
+          {% else %}
+            No tags yet. <a href="https://www.courtlistener.com{% url 'tag_favorites_help' %}">Learn more</a>
+          {% endif %}
+        </span>
+      </p>
+    {% else %}
+      <p style="font-weight: inherit; font-family: inherit; border: 0; vertical-align: baseline; font-style: inherit; margin: 1em 0 1em; padding: 0;">
+        <strong>Your Note and Tags: </strong><span>You have not added a notes or tags to this alert. <a href="https://www.courtlistener.com{% url 'tag_favorites_help' %}">Learn more</a></span>
+      </p>
```

Comment:
I think we can simplify in this case:

```suggestion
      <p style="font-weight: inherit; font-family: inherit; border: 0; vertical-align: baseline; font-style: inherit; margin: 1em 0 1em; padding: 0;">
        Use notes and tags to organize and share the cases you follow. <a href="https://www.courtlistener.com{% url 'tag_favorites_help' %}">Learn more</a>
      </p>
```

Reply:
Changed.